pkg/controller/stacks/install/installjob_test.go

```diff
@@ -418,6 +419,54 @@ func withJobSource(src string) jobModifier {
 	}
 }
 
+// withJobPullPolicy sets the image pull policy on the InitContainer, whose
+// container is derived from the stack. The non-init containers are not affected
+// since they reflect the imagePullPolicy of the stack-manager itself
+func withJobPullPolicy(pullPolicy corev1.PullPolicy) jobModifier {
+	return func(j *batchv1.Job) {
+		ics := j.Spec.Template.Spec.InitContainers
+		for i := range ics {
+			ics[i].ImagePullPolicy = pullPolicy
+		}
+	}
+}
+
+// withJobExpectations sets default job expectations
+func withJobExpectations() jobModifier {
+	return func(j *batchv1.Job) {
+		zero := int32(0)
+		j.SetResourceVersion("1")
+		j.SetGroupVersionKind(batchv1.SchemeGroupVersion.WithKind("Job"))
+		j.SetLabels(stacks.ParentLabels(resource()))
+		j.Spec.BackoffLimit = &zero
+		j.Spec.Template.Spec.Volumes = []corev1.Volume{{
+			Name:         "package-contents",
+			VolumeSource: corev1.VolumeSource{EmptyDir: &corev1.EmptyDirVolumeSource{}},
+		}}
+		j.Spec.Template.Spec.InitContainers = []corev1.Container{{
+			Name:         "stack-copy-to-volume",
+			Command:      []string{"cp", "-R", "/.registry", "/ext-pkg/"},
+			VolumeMounts: []corev1.VolumeMount{{Name: "package-contents", MountPath: "/ext-pkg"}},
+		}}
+		j.Spec.Template.Spec.Containers = []corev1.Container{
+			{
+				Name:  "stack-unpack-and-output",
+				Image: stackPackageImage,
+				Args: []string{
+					"stack",
+					"unpack",
+					"--content-dir=/ext-pkg/.registry",
+					"--permission-scope=Namespaced",
+					"--templating-controller-image=",
```

Comment:
Do we need a value for this parameter?

Reply:
In the test environment we would only have a value here if we supplied it in the harnessing or fixture.  This particular set of tests isn't concerned with that value.  This is the first place that I am aware of where we are checking the final shape of an install job, so we may revisit this in the future as we add more tests and composition helpers (`withTemplatingControllerImage()`).